test/api/unit/models/group.test.js

```diff
@@ -532,8 +539,10 @@ describe('Group Model', () => {
           let [
             updatedLeader,
             updatedParticipatingMember,
+            updatedSleepingParticipatingMember,
```

Comment:
`updatedSleepingParticipatingMember ` matches the third element array but `User.findById(sleepingParticipatingMember._id), ` is the second, not third element

Reply:
Ah yeah, you're right. It wouldn't have caused the test to fail since all three members should have the same data but it was definitely wrong and inconsistent. I've pushed a fix. Thanks!